orderbook/orderbook.go

```diff
@@ -5,3 +5,11 @@ import "fmt"
 func ID(baseID, quoteID string) string {
 	return fmt.Sprintf("%s-%s", baseID, quoteID)
 }
+
+type Side string
+
+const (
+	Unspecified Side = "unspecified"
```

Comment:
Will this allow us to create orders with Unspecified side? Not sure we want that...

Reply:
Yeah, there was a reason, but a poor one, I have removed it and will refactor the bit that needs it.